### PR TITLE
Documentating the new geolocation update feature

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- 100% tests passed

### CODE REVIEW

**Comment by XavierVal** [comment]
LGTM (passing the ball to @dmoranj)

File: README.md [hunk 1]
```diff
@@ -19,56 +19,201 @@ The Context Adapter is in charge of adapting the communication (protocols and me
...
+                                  "type": "string",
+                                  "value": &lt;mobile-network-code&gt;
+                                },
+                                {
+                                  "name": "lac",
+                                  "type": "string",
+                                  "value": &lt;location-area-code-as-hexadecimal-number&gt;
+                                },
+                                {
+                                  "name": "cell-id",
```
Comment (AlvaroVega) [inline README.md:387]: `cell-id` or `cellid`?

**Comment by AlvaroVega** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (gtorodelvalle) [inline README.md:387]: As far as I know, it is `cell-id` but pinging @mrutid for confirmation... :)

**Comment by dmoranj** [comment]
Please reorganize it using subsections, stating clearly: how to use the CA in the Black Button flow and how to use the CA for GPS translation.

**Comment by dmoranj** [comment]
I also think more literature is needed and less payloads (as some of them will never be seen by the user).

**Comment by gtorodelvalle** [comment]
Nice suggestion! :) I included the requested information and structure but maintained the payloads since I guess it adds value to the developer or even tester... :)

**Comment by dmoranj** [comment]
Now it looks great. Thanks.

LGTM

**Pull request merged**